src/content/decentralized-social-networks.md

```diff
@@ -0,0 +1,89 @@
+
+The Ethereum network has become the preferred tool for developers creating decentralized social media networks owing to the popularity of its tokens (ERC-20/ERC-721) and its massive user base. Here are some examples of Ethereum-based social networks:
+
+- Peepeth
+
+[Peepeth](https://peepeth.com/) is a microblogging platform similar to Twitter. It runs on the Ethereum blockchain and uses IPFS to store user data.
+
+Users can send short messages called "Peeps", which cannot be deleted or modified. You can collect tips or tip anyone on the platform in ether (ETH) without leaving the app.
+
+- Mirror
+
+[Mirror](https://mirror.xyz/) is a decentralized and user-owned content distribution platform similar to Medium. Users can sign up by connecting their wallets or providing an Ethereum Name Service (ENS) username.
+
+Posts published on Mirror are permanently stored on Arweave, a decentralized storage platform. Users can earn money by selling their articles as [non-fungible tokens (NFTs)](https://ethereum.org/en/nft/).
+
+- MINDS
+
+[MINDS](https://www.minds.com/) is one of the most used decentralized social media platforms. It works like Facebook and has racked up millions of users already.
+
+Users use the platform's native ERC-20 token $MIND to pay for items. Users can also earn $MIND tokens by publishing popular content, contributing to the ecosystem, and referring others to the platform.
+
+## Ethereum and legacy social media platforms {#Ethereum-and-legacy-social-media-networks}
+
+Decentralized social media platforms aren't the only ones trying to incorporate blockchain technology. Many traditional sites are also planning to integrate Ethereum into their infrastructure:
+
+**Reddit**
+
+Reddit has [touted Community Points](https://cointelegraph.com/news/reddit-to-reportedly-tokenize-karma-points-and-onboard-500m-new-users), which are ERC-20 token that users can earn by posting quality content and contributing to the online community. You can redeem these tokens to gain exclusive privileges and perks. For this project, Reddit is working with Arbitrum, a Layer-2 rollup designed to scale Ethereum transactions.
+
+**Twitter**
+
+In January 2021, Twitter Blue [rolled out support for NFTs](https://mashable.com/article/twitter-blue-nft-profile-picture), allowing users to connect their wallets and display NFTs as profile pictures. The social media company has also [announced plans](https://www.theverge.com/2021/8/16/22627435/twitter-bluesky-lead-jay-graber-decentralized-social-web) to create a decentralized social network in future.
+
+**Instagram**
+
+In May 2022, Instagram, a photo and video sharing social networking service with over a billion active monthly users, [announced support for NFTs](https://about.instagram.com/blog/announcements/instagram-digital-collectibles) on the Ethereum network and Polygon, a sidechain for Ethereum. Once they connect a wallet, users will be able to post and share NFTs on their Instagram profile. 
```

Comment:
- Let's assume users know of Instagram and its popularity
- Slight structure reorg of the last sentence to put the subject first

```suggestion
In May 2022, [Instagram announced support for NFTs](https://about.instagram.com/blog/announcements/instagram-digital-collectibles) on Ethereum and Polygon. Users can post NFTs directly to Instagram by connecting their Ethereum wallet. 
```